contact.html

```diff
@@ -3,6 +3,6 @@
 layout: default
 ---
       <div class="alert alert-info">
-        <strong>Have your own IRC client?</strong> You can chat with us on <a href="irc://irc.freenode.net/ircv3"><strong>irc.freenode.net</strong></a> in <a href="irc://irc.freenode.net/ircv3"><strong>#ircv3</strong></a>.
+        <strong>Have your own IRC client?</strong> You can chat with us on <a href="ircs://irc.freenode.net:6697/#ircv3"><strong>irc.freenode.net</strong></a> in <a href="ircs://irc.freenode.net:6697/#ircv3"><strong>#ircv3</strong></a>.
```

Comment:
`2015-09-26 16:32:48+0300 < Zarthus> Mikaela: re ircv3: turns out you're more right than I was. I've been using irc://irc.network.net/channel forever since my client detects irc://irc.network.net/#channel as ##channel`